scanner/cmd/scannerctl/vm.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	_ "embed"
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/stackrox/rox/pkg/jsonutil"
+	"github.com/spf13/cobra"
+	v4 "github.com/stackrox/rox/generated/internalapi/scanner/v4"
+)
+
+const mockDigest = "registry/repository@sha256:deadb33fdeadb33fdeadb33fdeadb33fdeadb33fdeadb33fdeadb33fdeadb33f"
+
+//go:embed fixtures/indexContents.json
+var contentsJson []byte
+
+// scanCmd creates the scan command.
+func scanVM(ctx context.Context) *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "vm",
+		Short: "Perform vulnerability scans.",
+		Args:  cobra.ExactArgs(0),
```

Comment:
Could we maybe accept the file as a parameter and take the embedded example only if the param is skipped?

Reply:
done

Reply:
Oh, I was thinking about changing it simply to 
```suggestion
		Args:  cobra.ExactArgs(1),
```
but a flag is also fine.